packages/tools/components-package/nps.js

```diff
@@ -1,15 +1,17 @@
 const path = require("path");
 const fs = require("fs");
 const LIB = path.join(__dirname, `../lib/`);
-const FIORI = path.join(__dirname, `../../fiori/`);
-
 
 const getScripts = (options) => {
 
-	let illustrations = options.illustrationsData || [];
+	let illustrations = illustrationsImports = options.illustrationsData || [];
 	illustrations = illustrations.map(illustration => `node "${LIB}/create-illustrations/index.js" ${illustration.path} ${illustration.defaultText} ${illustration.illustrationsPrefix} ${illustration.set} ${illustration.destinationPath}`);
 	let illustrationsScript = illustrations.join(" && ");
 
+	// create file with all illustrations' imports
+	illustrationsImports = illustrationsImports.map(illustrations => `node ${LIB}/generate-js-imports/illustrations.js ${illustrations.destinationPath.replace("/tnt", "")} ${illustrations.destinationPath} dist/generated/js-imports`);
+	let illustrationsImportsScript = illustrationsImports.join(" && ");
```

Comment:
I think that with this change:
- we will now call the generation script twice:
"
node tools/lib/generate-js-imports/illustrations.js dist/illustrations dist/illustrations dist/generated/js-imports 
&& 
node tools/lib//generate-js-imports/illustrations.js dist/illustrations dist/illustrations/tnt dist/generated/js-imports
"

- and the script is design to generate both fiori and tnt assets at one pass
- and checking the arguments for the first case we have "dist/illustrations dist/illustrations dist/generated/js-imports " first and second args are duplicated - not major issue, but strange

could you check if you can optimize it